services/order.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime
+) -> None:
+    order = Order.objects.create(username=username)
```

Comment:
Code style issue: It's better to use `get_user_model().objects.create_user()` instead of `Order.objects.create(username=username)`. This is a best practice.